news/2024/2024-09-11-osumania-4k-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,119 @@
+- [Ushirokara Haiyoritai G - Koi wa Chaos no Shimobenari](https://osu.ppy.sh/beatmapsets/2242646#mania/4766766) ([0DZ0](https://osu.ppy.sh/users/28156707), RC5) @ [30:33](https://www.twitch.tv/videos/2245925227?t=00h30m33s) - This point was a masterclass, and the UK was the teacher. A short and sweet technical map, as seen towards the weekend a single miss here means so much. Hong Kong misses. The UK doesn't. They played this map 2 times in this weekend. Those are the top 2 team scores out of every team in the tournament. Enough said.
+- [Kou! - Frost Walker](https://osu.ppy.sh/beatmapsets/2242668#mania/4766814) ([guden](https://osu.ppy.sh/users/11626065), TB) @ [62:40](https://www.twitch.tv/videos/2245925227?t=01h02m40s) - From a 5–2 lead, and after a tragic breakpoint in the chordjack pick, the UK finds themselves on 5–5. The devastating array of patterns laid out in this collab ensues, and after the dust settles, some of the LN patterns proved to be too much for the Brits, leading to a pretty narrow 7900 points loss. Actually, this is the second tiebreaker each of these teams play in the tournament — and their positions seem clear. 2 wins for Hong Kong. 2 losses for the United Kingdom.
+
+Commiserations to the British team for their run this year, unfortunately ended by 2 insanely close matches. Congratulations to Hong Kong for fighting on against all odds, next weekend they will face the Philippines in hopes of keeping it alive once more.
+
+## Score highlights
+
+There's no denying that all teams thus far have given their all, especially as we draw ever so closer to the lattermost rounds of the tournament. But this round's standout amongst all teams must've been Japan!
+
+- ::{ flag=JP }:: [[Crz]yomogi237](https://osu.ppy.sh/users/28571440) took everyone by surprise with their crazy ratio in [cygnus - Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640#mania/4766759) by [Miaurichesu](https://osu.ppy.sh/users/10055648) (RC3), achieving a score of 999,792 with only **10 perfects** in their match [against Spain](https://www.twitch.tv/videos/2244890379).
+- ::{ flag=JP }:: [CrewK](https://osu.ppy.sh/users/11488604) covered Japan's SV needs with their astonishing 997,856 score on [Tanger - BREAKME](https://osu.ppy.sh/beatmapsets/2242663#mania/4766804) by [Promachos](https://osu.ppy.sh/users/14069486) (SV1) in their match [versus Brazil](https://www.twitch.tv/videos/2245937607).
+
+Will they be able to hold up next week? Will the other teams keep their edge? Whatever the case may be, Japan has proved that they are a force to be reckoned with.
+
+---
+
+Tune in next week for the Semifinals recap, or even better, watch the matches live on the [osu!live Twitch channel](https://www.twitch.tv/osulive)! As always, you can browse the [official wiki page](/wiki/Tournaments/MWC/2024_4K), check out the schedule along with the full tournament details, and pick which teams you want to support in their quest to become champions.
```

Comment:
i usually want to omit using "official" because in most cases it's not necessary to disambiguate anything and is therefore meaningless

```suggestion
Tune in next week for the Semifinals recap, or even better, watch the matches live on the [osu!live Twitch channel](https://www.twitch.tv/osulive)! As always, you can browse the [wiki page](/wiki/Tournaments/MWC/2024_4K), check out the schedule along with the full tournament details, and pick which teams you want to support in their quest to become champions.
```